.github/auto_assign.yml

```diff
@@ -2,7 +2,7 @@
 # https://github.com/kentaro-m/auto-assign/
 
 # Set to true to add reviewers to pull requests
-addReviewers: true
+addReviewers: false
```

Comment:
Does this file do anything now? Can we just remove the whole thing?

Reply:
Zach decided to punt on landing this just yet, I'll close it out.  FWIW I thought disabling the bot via configuration files was ideal to poking people to kill it.